lib/msteams_hermes/components/adaptive_card.rb

```diff
@@ -14,7 +14,12 @@ module Components
     class AdaptiveCard < Base
       attr_reader :body, :actions, :entities, :width
 
-      def initialize(body:, actions: nil, entities: [], full_width: false)
+      WIDTH = {
+        default: "default",
```

Comment:
In ruby, when you want to create a constant, ruby encourages to capitalize them.